### ISSUE TITLE
UI improvement: gather tools, colors and layers

### ISSUE THREAD

**Issue by StefanoCecere** [guillaumechereau/goxel#69]
hi.. i think that Goxel has a lot of potential.. but it's quite unusable from the UX point of view!
it's very difficult to change panels with the mouse,.. and since there's a lot of space, the basic tools, the color palette and the Layers could all be displayed together, no?

at least map them to some keyboard shortcut for fast switch

**Comment by guillaumechereau** [comment]
Yeah it would be nice to support multiple pans in the UI that we can move around.  I am busy at the moment, but I will try to do something like that at some point.
